result/bgpv4.yaml

```diff
@@ -24,35 +24,89 @@ components:
           description: >-
             The list of column names that the returned result set will contain.
             If the list is empty then all columns will be returned except for
-            any result_groups.
-            The name of the BGPv4 device cannot be excluded.
+            any result_groups. The name of the BGPv4 peer cannot be excluded.
           type: array
           items:
             type: string
             enum:
               - session_state
-              - routes_advertised
-              - routes_withdrawn
+              - flap_count
+              - routes_sent
+              - routes_received
+              - route_withdraws_sent
+              - route_withdraws_received    
```

Comment:
is there a difference between `routes_sent` and `route_advertisements_sent` ?

if latter is more appropriate, then we should be consistent with naming; route_<plural noun>_<sent/received>
- route_advertisements_sent
- route_advertisements_received
- route_withdrawals_sent
- route_withdrawals_received

Reply:
Second does not look right due to confusion between routes_withdrawn and routes_removed/deleted. First one is closer to what we want but routes_sent feels more appropriate since "advertisements" and "sent" have similar meaning. Hence the short name is better.